src/main/java/org/example/springbootintro/controller/BookController.java

```diff
@@ -5,15 +5,19 @@
 import org.example.springbootintro.dto.book.BookDto;
 import org.example.springbootintro.dto.book.CreateBookRequestDto;
 import org.example.springbootintro.service.book.BookService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping(value = "api/books")
+@RequestMapping(value = "apibooks")
```

Comment:
You can leave '/books'. And '/api' move to application.properties `server.servlet.context-path=/api`